### ISSUE TITLE
Cannot redact the text

### ISSUE THREAD

**Issue by Winchester80** [zendesklabs/ticket_redaction_app#79]
in a ticket, i've this problem: 
![image](https://user-images.githubusercontent.com/38953238/39583853-53979c1c-4ef1-11e8-9ddb-18d10a97c026.png)

**Comment by dpawluk** [comment]
@Winchester80 are you using rich text? Can you redact the two pieces separately not including the newline?

**Comment by Winchester80** [comment]
Hi, yes i try the two pieces not including the newline but don’t work anyway

 

Da: Daniel Pawluk [mailto:notifications@github.com] 
Inviato: lunedì 7 maggio 2018 05:26
A: zendesklabs/ticket_redaction_app <ticket_redaction_app@noreply.github.com>
Cc: Winchester80 <diego.petruzzino@sintesys.it>; Mention <mention@noreply.github.com>
Oggetto: Re: [zendesklabs/ticket_redaction_app] Cannot redact the text (#79)

 

@Winchester80 <https://github.com/Winchester80>  are you using rich text? Can you redact the two pieces separately not including the newline?

**Comment by dpawluk** [comment]
are their special characters in the text that needs to be redacted? Anything out of the ascii range. I realize that since it is being redacted its too sensitive to share on a public forum. Maybe its possible to redact the text in as small of segments as possible to figure out if there is a specific character that is causing problems?

**Comment by AleksanderMN** [comment]
We too cannot redact all text, getting same error. I believe the cause is the Nordic letters Æ-Ø-Å. Can you fix that?

**Comment by Winchester80** [comment]
I do not think it depends on a problem of special characters because in the text I'm trying to redact, there are not any. In the last attempt I selected a simple "ok" with the same result

**Comment by brendapiersak** [comment]
We are experiencing the same issue.  No special characters, it will not redact any text from the ticket.
